_posts/2019-10-19-Slamdunk.md

```diff
@@ -0,0 +1,31 @@
+---
+site: vienna
```

Comment:
there is no site vienna, you could become an official "member site", see here: https://galaxyproject.eu/freiburg/

But until then this does not work, can you change this to "freiburg"?